### ISSUE TITLE
update Metals version to the newest one through dialogue only updates in User tab.

### ISSUE THREAD

**Issue by vzmerr** [scalameta/metals-vscode#1071]
### Describe the bug

To reproduce, put `0.11.8-SNAPSHOT` in Metals Server Version in both Workspace and User tabs of VSCode. Then, totally close VSCode. (Do not `Relead Window` instead.) And reopen it. 
A dialogue would appear asking if you want to update the Metals Version to the newest one. And you should confirm. Then reload and check the Metals Version setting:
It is only updated in the User tab. But remains as `0.11.8-SNAPSHOT` in the Workspace tab, which overrides the User one.

### Expected behavior

choosing to update the Metals version to the newest one, through the dialogue in VSCode, should change the setting also in the Workspace tab for the result to be effective.

### Operating system

macOS

### Editor/Extension

VS Code

### Version of Metals

v0.11.8-SNAPSHOT

**Comment by tanishiking** [comment]
I transferred it to `metals-vscode`, since this is a problem only with the VS Code extension :)

**Comment by tanishiking** [comment]
Indeed, metals-vscode updates only one configuration (of User and Workspace configurations)

- Get the configuration value and target from `User` **or** `Workspace`
  - https://github.com/scalameta/metals-vscode/blob/1822de60091af184725383b0476c7f883cbab53e/src/getServerVersion.ts#L33-L36
- Ask the user and update the config
  - https://github.com/scalameta/metals-vscode/blob/1822de60091af184725383b0476c7f883cbab53e/src/getServerVersion.ts#L57-L61

we might want to update both `User` and `Workspace` of the server version configuration if both of them has outdated version 🤔
